python/podio_class_generator.py

```diff
@@ -271,23 +310,33 @@ def _process_component(self, name, component):
 
     component['includes'] = self._sort_includes(includes)
     component['class'] = DataType(name)
-    self._fill_templates('Component', component)
-
-    # Add potentially older schema for schema evolution
-    # based on ROOT capabilities for now
-    if name in self.root_schema_dict:
-      schema_evolutions = self.root_schema_dict[name]
-      component = copy.deepcopy(component)
-      for schema_evolution in schema_evolutions:
-        if isinstance(schema_evolution, RenamedMember):
-          for member in component['Members']:
-            if member.name == schema_evolution.member_name_new:
-              member.name = schema_evolution.member_name_old
-          component['class'] = DataType(name + self.old_schema_version)
-        else:
-          raise NotImplementedError
+    component['upstream_edm'] = self.upstream_edm
+    component['upstream_edm_name'] = ''
+    if self.upstream_edm:
+      component['upstream_edm_name'] = self.upstream_edm.options["includeSubfolder"].split("/")[-2].capitalize()
+
+    if self.proglang == "julia":
+      self._fill_templates('MutableStruct', component)
+
```

Comment:
```suggestion
    return component
```
Can we return early here to avoid having a larger than necessary diff (from re-indenting due to the `if self.proglang == "cpp":`)?

Reply:
I indented the code `if self.proglang == "cpp":` so that it runs for c++ only and not for julia.